programs/mango-v4/src/accounts_ix/token_liq_bankruptcy.rs

```diff
@@ -40,6 +40,7 @@ pub struct TokenLiqBankruptcy<'info> {
 
     #[account(mut)]
     // address is checked at #2 a) and b)
+    // better name would be "insurance_bank_vault"
```

Comment:
imo, we should do announcements and remove unused ixs from program after newer versions are 2-3 releases behind to slim down idl

Reply:
Keeping them in the idl allows the explorer to still work nicely when browsing old transaction, that's why I would prefer to keep everything in the idl. But yeah, we do need to figure out growing the idl account above 10k soonish.